oap-server/server-bootstrap/src/main/resources/ui-initialized-templates/topology-service-relation.yml

```
@@ -120,127 +120,83 @@ templates:
         },
-        "tcp":{
+        "envoy_metrics":{
           "server":[
+          ],
+          "client":[
             {
               "width":12,
-              "title":"Response Time",
+              "title":"Upstream CX Active",
               "height":"200",
               "entityType":"ServiceRelation",
               "metricType":"REGULAR_VALUE",
-              "metricName":"service_relation_server_resp_time",
+              "metricName":"envoy_sr_cluster_up_cx_active",
               "queryMetricType":"readMetricsValues",
-              "chartType":"ChartLine",
-              "unit":"ms"
+              "chartType":"ChartLine"
             },
             {
               "width":12,
-              "title":"Response Time Percentile",
-              "height":200,
-              "entityType":"ServiceRelation",
-              "metricType":"LABELED_VALUE",
-              "metricName":"service_relation_server_percentile",
-              "queryMetricType":"readLabeledMetricsValues",
-              "chartType":"ChartLine",
-              "unit":"ms",
-              "metricLabels":"P50, P75, P90, P95, P99",
-              "labelsIndex":"0, 1, 2, 3, 4"
-            },
-            {
-              "width":12,
-              "title":"Successful Rate",
+              "title":"Upstream CX Increase",
               "height":"200",
               "entityType":"ServiceRelation",
               "metricType":"REGULAR_VALUE",
-              "metricName":"service_relation_server_call_sla",
+              "metricName":"envoy_sr_cluster_up_cx_incr",
               "queryMetricType":"readMetricsValues",
               "chartType":"ChartLine",
-              "unit":"%",
-              "aggregation":"/",
-              "aggregationNum":"100"
+              "unit":"Per Minute"
             },
             {
               "width":12,
-              "title":"Load",
+              "title":"Upstream RQ Active",
               "height":"200",
               "entityType":"ServiceRelation",
               "metricType":"REGULAR_VALUE",
-              "metricName":"service_relation_server_cpm",
+              "metricName":"envoy_sr_cluster_up_rq_active",
               "queryMetricType":"readMetricsValues",
-              "chartType":"ChartLine",
-              "unit":"PPM - Packets Per Minute"
+              "chartType":"ChartLine"
             },
             {
               "width":12,
-              "title":"Throughput",
-              "height":"200",
-              "entityType":"ServiceRelation",
-              "metricType":"REGULAR_VALUE",
-              "metricName":"service_relation_server_received,service_relation_server_sent",
-              "queryMetricType":"readMetricsValues",
-              "chartType":"ChartLine",
-              "unit":"PPM - Packets Per Minutes"
-            }
-          ],
-          "client":[
-            {
-              "width":12,
-              "title":"Response Time",
+              "title":"Upstream RQ Increase",
```

You already use abbreviations in the metrics names, let's use the full words in the title, otherwise there is nowhere to know the full description, the same for other places

```suggestion
              "title":"Upstream Request Increase",
```